### ISSUE TITLE
Add more examples of arrays to the Cookbook

### ISSUE THREAD

**Issue by novusnota** [tact-lang/tact#1140]
As shown here https://github.com/tact-lang/tact/issues/163, we can make something like [bit arrays](https://en.wikipedia.org/wiki/Bit_array), but allow people to store numbers as byte offsets into the Int type and add subsequent helper wrappers to operate on such "array".

To not pollute global Int method space, we could wrap them into their own struct, like: `struct Array32 { u8: Int }`.

Additional methods/extension functions could be added, like counting the sum of bytes in the array and such.

P.S.: Solidity has a value/primitive type called [Fixed-size byte arrays](https://docs.soliditylang.org/en/v0.8.25/types.html#fixed-size-byte-arrays), which is similar in spirit to what we're trying to do here, so we might just call our arrays the same.

**Comment by anton-trunov** [comment]
I would propose to use a different name for the underlying integer, because `u8` for a 257-bit integer sounds confusing :)

And this is a perfect showcase for struct fields with default values

**Comment by novusnota** [comment]
> I would propose to use a different name for the underlying integer, because u8 for a 257-bit integer sounds confusing :)

Hmm, this could be better: `struct Uint8Array { arr32: Int }`

Inspiration is taken from: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array

**Comment by anton-trunov** [comment]
> Additional methods/extension functions could be added, like counting the sum of bytes in the array and such.

yeah, the sum of bytes can be made much more efficient than just looping and using `.get`
